Parser/pegen_errors.c

```diff
@@ -259,15 +259,15 @@ get_error_line_from_tokenizer_buffers(Parser *p, Py_ssize_t lineno)
     const char* buf_end = p->tok->fp_interactive ? p->tok->interactive_src_end : p->tok->inp;
 
     for (int i = 0; i < relative_lineno - 1; i++) {
-        char *new_line = strchr(cur_line, '\n') + 1;
```

Comment:
`NULL + 1` is beautifully strongly typed, huh? 😎

Reply:
![6lujhh.jpg](https://user-images.githubusercontent.com/11718525/177373425-49d95bc0-bc54-498d-8dea-d7fd7f456b3a.jpg)